app/javascript/react/components/Artists/ArtistTile.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import {Link} from "react-router-dom";
+
+const ArtistTile = (props) => {
+    
+    return (
+    <div className="artist-tiles">
+        <div className="small-3">
+            <div className="card" style={{width: 300}}> 
+                <img src="http://cache.boston.com/resize/bonzai-fba/Globe_Photo/2011/04/14/1302796985_4480/539w.jpg"/>
```

Comment:
if you havent made a feature that allows a user to add a photo, this shouldnt be present. No dummy data unfortunately.